### ISSUE TITLE
cannot install purescript on node docker images

### ISSUE THREAD

**Issue by FloWi** [purescript/purescript#4108]
## Description

Hi! 

Today I created a new purescript app and unfortunately it didn't work in a docker-image like the ones before.

The installation inside a docker container fails. We use the prebuilt node-images from dockerhub. I upgraded to the latest version `node:16-buster`, but the error still persists.

```
...
npm ERR! code 1
npm ERR! path /build/node_modules/purescript
npm ERR! command failed
npm ERR! command sh -c install-purescript --purs-ver=0.14.2
npm ERR! [ SUCCESS ] Check if a prebuilt 0.14.2 binary is provided for linux (574ms)
npm ERR! [ SUCCESS ] Download the prebuilt PureScript binary (2s)
npm ERR! [ FAILURE ] Verify the prebuilt binary works correctly
npm ERR! Error: Command failed: /build/node_modules/purescript/purs.bin --version
npm ERR! /build/node_modules/purescript/purs.bin: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /build/node_modules/purescript/purs.bin)
npm ERR!
npm ERR!     at ChildProcess.exithandler (node:child_process:326:12)
npm ERR!     at ChildProcess.emit (node:events:394:28)
npm ERR!     at maybeClose (node:internal/child_process:1067:16)
npm ERR!     at Socket.<anonymous> (node:internal/child_process:453:11)
npm ERR!     at Socket.emit (node:events:394:28)
npm ERR!     at Pipe.<anonymous> (node:net:661:12)
npm ERR!
...
```
## To Reproduce

I created a minimal example with a Dockerfile. 

```shell
git clone git@github.com:FloWi/purescript-installation-broken-minimal.git
docker build --tag purescript-installation-broken-minimal  .
```
## Expected behavior

a working installation of purescript.

## Additional context

I did some research and found out that the node images use a slightly older GLIBC version than the one you require now.

```shell
➜ docker run --rm -ti node:15-buster bash
root@02515c2a0b5b:/# ldd --version
ldd (Debian GLIBC 2.28-10) 2.28
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
root@02515c2a0b5b:/#
```

I haven't found a way to upgrade glibc and unfortunately I'm no expert in the node-ecosystem. Do you have a recommendation for the environment to build and run purescript apps? We used the prebuild node images to avoid installing it manually. Some tips would be appreciated.

## PureScript version

0.14.2

**Comment by kritzcreek** [comment]
This sounds related to: https://discourse.purescript.org/t/dockerfile-for-purescript-0-14-2/2379/2

**Comment by hdgarrood** [comment]
Thanks for the report! Yes, this will be addressed by #4107, so I think I'm going to close this so that we can track the work there.

**Comment by FloWi** [comment]
Ah, you're already aware of the issue. Sorry for the duplicate- haven't found the other issue. 

Thanks for your work!
